install/nightly/InstallHalyard.sh

```diff
@@ -1,427 +1,23 @@
 #!/usr/bin/env bash
 
-# This script installs Halyard.
+TEMPDIR=$(mktemp -d installhalyard.XXXX)
 
-# _WARNING_ This file was auto generated, do not edit directly
+pushd $TEMPDIR
 
-set -e
-set -o pipefail
+echo "Downloading nightly halyard installer..."
 
-REPOSITORY_URL="https://dl.bintray.com/spinnaker-releases/debians"
-SPINNAKER_REPOSITORY_URL="https://dl.bintray.com/spinnaker-releases/debians"
-SPINNAKER_DOCKER_REGISTRY="gcr.io/spinnaker-marketplace"
-SPINNAKER_GCE_PROJECT="marketplace-spinnaker-release"
+curl -O https://raw.githubusercontent.com/spinnaker/halyard/master/install/Installer.sh
```

Comment:
No switch being flipped here, just switching to a single install script for the old path rather than two that are written from a template. The goal was just to lower the cognitive overhead required to understand all the different new scripts in the `./install` directory.